Multisig/UI/Transaction/TransactionListViewController/Cells/LoadingFooterView.xib

```diff
@@ -22,7 +22,7 @@
                             <rect key="frame" x="197" y="20" width="20" height="22"/>
                         </activityIndicatorView>
                     </subviews>
-                    <color key="backgroundColor" name="backgroundPrimary"/>
+                    <color key="backgroundColor" name="backgroundContent"/>
```

Comment:
Will this affect some other tables in other screens? (similar comment to other footers). Why is the color changed? i.e. was it wrong?

Reply:
yes, content background is different from primary background
![image](https://user-images.githubusercontent.com/17750984/205619716-63aab06f-0cd8-4e20-9c23-fefaa17fe09a.png)

Reply:
ok, so before it was showing a visible horizontal band?

Reply:
exactly